File: experiment_jp_test/mutants/if__0__FIND_MAXIMUM_AVERAGE_SUBARRAY_OF_K_LENGTH.java
static int findMaxAverage ( int [ ] arr , int n , int k ) { int [ ] csum = new int [ n ] ; csum [ 0 ] = arr [ 0 ] ; int max_sum = csum [ k - 1 ] , max_end = k - 1 ; return max_end - k + 1 ; }

File: experiment/java-python/test/j2pyrawpy/if__0__FIND_MAXIMUM_AVERAGE_SUBARRAY_OF_K_LENGTH.py
#!/usr/bin/env python
""" generated source for module if__0__FIND_MAXIMUM_AVERAGE_SUBARRAY_OF_K_LENGTH """
class X(object):
    """ generated source for class X """
    @classmethod
    def findMaxAverage(cls, arr, n, k):
        """ generated source for method findMaxAverage """
        csum = [None]*n
        csum[0] = arr[0]
        max_sum = csum[k - 1]
        max_end = k - 1
        return max_end - k + 1

